src/controllers/deploy.go

```diff
@@ -79,7 +84,14 @@ func internaldeploy(a *history.ActionInstance) ([]byte, error) {
 			return output, err1
 		}
 
-		args := GetDeployArgs(a.RepoURL, branch, a.Server, a.Subdomain, a.Access)
+		kwargs := make(map[string]bool)
+		switch params.command {
+		case "redeploy":
+			kwargs["redeploy"] = true
+		}
+
+		args := getDeployArgs(a.RepoURL, branch, a.Server, a.Subdomain, a.Access, kwargs)
```

Comment:
If a service is in `stopped` state then its nginx entry does not exist. Adding the `x` flag here would mean that no nginx entry will be present for the service if a `redeploy` action is called in `stopped` state.

Reply:
Also, in this current logic a redeploy will never happen in a `running` state which shouldn't be the case.